index.html

```diff
@@ -14,14 +14,14 @@
 
 <body>
     <main class="main_content">
-        <section class="header">
-            <nav class="menu-icon">
-                <ul>
-                    <a href="index.html"><img src="images/icons/logo.png" alt="Logo" /></a>
-                    <li class="menu-item2"><img src="images/icons/Union.png" alt="Main menu"></li>
-                </ul>
+        <header class="header">
+            <div class="menu-icon">
+                <a href="index.html"><img src="images/icons/logo.png" alt="Logo" /></a>
+            <nav>
+                <img src="images/icons/Union.png" alt="Main menu">
             </nav>
-        </section>
+            </div>
+        </header>
         <section class="headline">
             <h1 class="headline-title">
                 Hey There<br />
```

Comment:
**[OPTIONAL BUT REQUIRED]**


- Hey, great job resolving the issue raised by the previous reviewer, however, your code should look like the code snippet given by the previous reviewer. You've wrapped the header inside of the main tag which is not a good practice and a good use of the semantic tag in your page layout. please, kindly remove the header from the tag. see this [link](https://stackoverflow.com/questions/20470345/should-the-header-and-footer-tags-be-inside-the-main-tag) for more details.

NOTE: *Please, fix this before you merge as this would be checked for your next project milestone. It's best if you fixed it now* :pray: